app/models/restroom/yelp.rb

```diff
@@ -0,0 +1,33 @@
+require Rails.root.join('lib', 'yelp_client')
+class Restroom::Yelp
+  class << self
+    def near(lat, lng)
+      conn = Faraday.new(:url => "https://api.yelp.com/v3/businesses/search?latitude=#{lat}&longitude=#{lng}&attributes=gender_neutral_restrooms")
+      results = conn.get do |req|
```

Comment:
Do we need to catch network errors here? We can log an error and return an empty array in that case

Reply:
Yes network errors are to be traced for ascertaining that the code problem is not there and only network issues are there.
This happens with high performance media sites like http://vov.media as well that despite everything perfect, network errors at places create function hindrances.

Need to address things duly that we  do not beat behind the bush and jump up to dry he conclusion promptly.